index.js

```diff
@@ -40,6 +40,16 @@ function AvaFiles(options) {
 		files = defaultIncludePatterns();
 	}
 
+	files = files.map(function (file) {
```

Comment:
I'd move this to line 37 and change it to:

```js
var files = (options.files || []).map(
```

Then remove the `!files ||` part of the condition on line 39 to assign the default patterns. Those don't start with `./` so don't need to be mapped. Focusing the mapping on the `files` that are actually provided seems clearer.